src/components/ui/MasqAlert.jsx

```diff
@@ -0,0 +1,41 @@
+/* global _ */
+import React, { useState } from 'react';
+import nconf from '@qwant/nconf-getter';
+import Alert from 'src/components/ui/Alert';
+
+const masqAlertEnabled = nconf.get().masq.alertEnabled;
+const masqLink = nconf.get().masq.link;
+const masqAlertDate = (new Date(nconf.get().masq.alertDate)).toDateString();
+
+const MasqAlert = () => {
+  const masqDismissed = window.localStorage.getItem('masq_alert_dismissed');
```

Comment:
Is there not a risk that `localStorage.getItem()`  raises an error if `localStorage` is not available ?  

It would also be safer to not use the localStorage at all if the alert is disabled by the configuration.